composer.lock

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
We don't necessarily need to upgrade Drupal now, the review should be to validate the approach.